packages/@react-aria/slider/docs/useSlider.mdx

```diff
@@ -0,0 +1,280 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/slider';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-aria/slider/package.json';
+import statelyDocs from 'docs:@react-stately/slider';
+import i18nDocs from 'docs:@react-aria/i18n';
+
+```jsx import
+import {useSlider, useSliderThumb} from '@react-aria/slider';
+```
+
+---
+category: Forms
+keywords: [slider, range, aria]
+---
+
+# useSlider
+
+<p>{docs.exports.useSlider.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useSlider', 'useSliderThumb']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/TR/wai-aria-1.2/#slider'}
+  ]} />
+
+
+## API
+
+<FunctionAPI function={docs.exports.useSlider} links={docs.links} />
+<FunctionAPI function={docs.exports.useSliderThumb} links={docs.links} />
+
+## Features
+
+The [&lt;input type="range"&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range)
+HTML element can be used to build a slider, however it is
+very difficult to style cross browser. `useSlider` and `useSliderThumb` help achieve accessible
+sliders that can be styled as needed.
+
+
+* Exposed to assistive technology as a `group` and `slider` via ARIA
+* Labeling support for accessibility
+* Internationalized number formatting as a percentage or value
+* Multiple thumbs, or group of range values
+* Horizontal and Vertical layouts
+* Normalized across devices
+
+## Anatomy
+
+Sliders consist of a track element showing the range of available values,
+a thumb or multiple thumbs showing the current value(s), and a label.
+Thumbs consist of an indicator that designates a position on the track.
+The track element represents the value(s) visually, the thumbs represent individual
+values with `role=slider`. The group of value(s) is represented by the wrapping element
```

Comment:
Maybe out of scope of this PR, but do we actually apply the "slider" role on the thumbs?

Reply:
kind of but not really, it's an input type="range"
but there is a case that will probably need to be covered as you said out of scope probably https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role